src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -369,7 +374,7 @@ protected void setup()
 
         // schedule periodic dumps of table size estimates into SystemKeyspace.SIZE_ESTIMATES_CF
         // set cassandra.size_recorder_interval to 0 to disable
```

Comment:
when we rename SIZE_RECORDER_INTERVAL key, this will stay unchanged. Better to somehow reference that SIZE_RECORDER_INTERVAL rather than mentioning here the property as is now here.

Reply:
Fixed.